xen/include/public/io/displif.h

```diff
@@ -38,7 +38,7 @@
  *                           Protocol version
  ******************************************************************************
  */
-#define XENDISPL_PROTOCOL_VERSION     "1"
+#define XENDISPL_PROTOCOL_VERSION     1
```

Comment:
This can break existing backends and frontends.
I think, only viable approach is to introduce a new define:
```
#define XENDISPL_PROTOCOL_VERSION_INT 1
#define XENDISPL_PROTOCOL_VERSION #XENDISPL_PROTOCOL_VERSION_INT
```

Reply:
well,
```
/tmp/vers.c: In function ‘main’:
/tmp/vers.c:4:35: error: stray ‘#’ in program
 #define XENDISPL_PROTOCOL_VERSION #XENDISPL_PROTOCOL_VERSION_INT

```
and we actually end up with some sort of expansions like

```
#define STR_HELPER(x) #x
#define STR(x) STR_HELPER(x)

#define XENDISPL_PROTOCOL_VERSION_INT 1
#define XENDISPL_PROTOCOL_VERSION STR(XENDISPL_PROTOCOL_VERSION_INT)

```
which is not really cute